From fce391236b063ad8674b3f01e3a14188263aba4d Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Tue, 16 Nov 2021 11:25:26 +0000
Subject: [PATCH 1248/2201] dtoverlays: Remove incorrect disable-touch fragment
 in vc4-kms-dsi-7inch

It referenced fragment@13 which used to be part of edt-ft5406.dtsi,
but has now been removed.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/vc4-kms-dsi-7inch-overlay.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/overlays/vc4-kms-dsi-7inch-overlay.dts b/arch/arm/boot/dts/overlays/vc4-kms-dsi-7inch-overlay.dts
index ecd3bef3d65a..5e1700d0367a 100644
--- a/arch/arm/boot/dts/overlays/vc4-kms-dsi-7inch-overlay.dts
+++ b/arch/arm/boot/dts/overlays/vc4-kms-dsi-7inch-overlay.dts
@@ -113,6 +113,6 @@
 	};
 
 	__overrides__ {
-		disable_touch = <0>, "-10-11-12-13";
+		disable_touch = <0>, "-10-11-12";
 	};
 };
-- 
2.17.1

